tutor/src/components/notes.js

```diff
@@ -366,6 +371,15 @@ class NotesWidget extends React.Component {
     }
   }
 
+  @action.bound setupModalScrollTop() {
+    let modalBody = window.document.body.querySelector('.modal-body');
+    this.modalScrollHelper = new ScrollTo({ windowImpl: modalBody, root: modalBody });
+  }
+
+  @action.bound onModalScollTop() {
+    this.modalScrollHelper.scrollToSelector('.filter-area', { updateHistory: false });
+  }
+
```

Comment:
I'm sorry, I hadn't realized that we were scrolling an inner div and not the entire window when you asked about scrolling earlier.  In this case you can remove the `setupModalScrollTop ` and use the  `scroll-into-view` library that we import from https://github.com/KoryNunn/scroll-into-view

an example of it is at: https://github.com/openstax/tutor-js/blob/master/tutor/src/components/book-menu/ux.js#L72

I think it'd look like:
```js
  @action.bound onModalScollTop() {
      scrollIntoView(document.querySelector('.modal-body'), { time: 300 });
   }
```

If the library scrolls things that shouldn't move then you'll have to use the validTarget option like the example in book-menu does.

Reply:
This is great and exactly what I needed, thanks!!